application/config/config.go

```diff
@@ -476,14 +476,10 @@ func (c *Config) ConfigureLogging(server lsp.Server) {
 	// env var overrides flag
 	envLogLevel := os.Getenv("SNYK_LOG_LEVEL")
 	if envLogLevel != "" {
-		fmt.Println("Setting log level from environment variable (SNYK_LOG_LEVEL)", envLogLevel)
```

Comment:
why not keep the messages and use `os.Stderr.WriteString` instead? That way we can keep the messages, but don't mess up stdout.

Reply:
Done. I'm using `fmt.Fprintln(os.Stderr, ...) ` to keep it shorter